### PR TITLE
Lrsd 4766 [Learn] Add page banner, course info tags, side nav , bottom nav and sign-in cta

### PR DESCRIPTION
**Pull request by luisacmn**

### CODE REVIEW

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-course-intro-bottom-nav/ddm-template.ftl [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+	<div>
+		<#if childrenArrayLength gt 0>
+			<ul class="m-0 p-2">
+				<#list 0..childrenArrayLength-1 as i>
+					<#assign child = childrenJSONArray.getJSONObject(i) />
+					<#if i = 1>
+						<li class="course-bottom-nav-item">
+							<a
+								class="d-flex justify-content-between" ${(navigationJSONObject.getJSONObject("self").url == child.url)?then("selected", "")}'
```
Comment (sammonsjl) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-course-intro-bottom-nav/ddm-template.ftl]: @luisacmn there appears to be an extra ' at the end of this line.  My editor is complaining about it so can you take a look?

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/sign-in/index.js
Comment (sammonsjl) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/sign-in/index.js:1]: @luisacmn yarn format is finding formatting issues with this file.

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-course-side-nav/test.json
Comment (sammonsjl) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-course-side-nav/test.json:1]: @luisacmn I just wanted to confirm that test.json is supposed to be committed.

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-course-intro-bottom-nav/ddm-template.ftl (hunk 1 above)
Comment (luisacmn) [reply workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-course-intro-bottom-nav/ddm-template.ftl]: Hey @sammonsjl . Sure, thanks.

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-course-side-nav/test.json
Comment (luisacmn) [reply workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-course-side-nav/test.json:1]: It's not. I'm removing it. Thanks

**Comment by luisacmn** [comment]
@sammonsjl Fixed!

**Comment by sammonsjl** [comment]
Thank you, see: https://github.com/brianchandotcom/liferay-portal/pull/151369

**Comment by luisacmn** [comment]
@sammonsjl resent!

**Comment by sammonsjl** [comment]
Thank you, resent here: https://github.com/brianchandotcom/liferay-portal/pull/151474

**Pull request closed without merging**